source/ScrollVar.h

```diff
@@ -94,15 +107,15 @@ bool ScrollVar<T>::Scrollable() const
 template <typename T>
 bool ScrollVar<T>::IsScrollAtMin() const
 {
-	return this->Value() >= T{};
+	return this->Value() <= T{};
```

Comment:
Yeah... The ScrollVar was outputting negative values, which was neat in that you could just add the value to the drawing offset, but bad because it meant that the value was flipped from what you thought it was... Thus the old logic would result in something like this:
```cpp
   ScrollVar<double> v;
   v = 5;
   double x = v;
   // x would be -5 here
```

I flipped all of that logic so that the offset and the values set are what you think they are now.